.mocharc.jsonc

```diff
@@ -0,0 +1,12 @@
+{
+	"diff": true,
+	"require": ["./tshook.js", "source-map-support/register"],
+	"extension": ["js", "ts"],
+	"package": "./package.json",
+	"recursive": true,
+	"reporter": "spec",
+	"slow": 75,
+	"timeout": 5000,
+	"ui": "bdd",
+	"exit": true
```

Comment:
What's this file added for?

Reply:
mochars file is a global config file for mocha. It sets stuff like timesouts, but alos hooks like "tshook.js"..this is needed to run typescript files during test.